tests/gorepo_test.go

```diff
@@ -5,11 +5,19 @@ package tests_test
 import (
 	"os"
 	"os/exec"
+	"strconv"
 	"testing"
 )
 
 // Go repository basic compiler tests, and regression tests for fixed compiler bugs.
 func TestGoRepositoryCompilerTests(t *testing.T) {
+	if b, _ := strconv.ParseBool(os.Getenv("SOURCE_MAP_SUPPORT")); os.Getenv("SOURCE_MAP_SUPPORT") != "" && !b {
+		t.Fatal("Source maps disabled, but required for this test. Use SOURCE_MAP_SUPPORT=true or unset it completely.")
```

Comment:
> Maybe skip if short flag is given?

I don't think that would be good, better be consistent.

I think I've seen `Skip` in the core packages for this situation. Reasoning might go like this:
Fail -> Implementation broken
Skip -> Environment broken/insufficient